seqr/views/apis/staff_api_tests.py

```diff
@@ -486,7 +486,7 @@ def test_mme_details(self, mock_datetime):
         url = reverse(mme_details)
         self.check_staff_login(url)
 
-        mock_datetime.now.return_value = datetime(2020, 4, 27, 20, 16, 01)
+        mock_datetime.now.return_value = datetime(2020, 4, 27, 20, 16, 1)
```

Comment:
`01` will cause the coverage tool to fail to collect coverage numbers for Python 3. Therefore, change it to `1`.